static/js/notifications.js

```diff
@@ -634,6 +630,11 @@ exports.notify_local_mixes = function (messages) {
             return;
         }
 
+        var link_msg_id = message.id;
+        var link_class = "compose_notification_narrow_by_topic";
+        var link_text = i18n.t("Narrow to __- message_recipient__",
+                               {message_recipient: get_message_header(message)});
+
```

Comment:
The reason this was ordered how it is is that we're likely to add a link to scroll to the message, which would result in us potentially moving some of these up.  Though maybe it's just the first one, so OK.

Reply:
yeah, I'd rather it be clean now, and we can always move them back (and we may have different values for the scroll case)